src/auspex/analysis/helpers.py

```diff
@@ -17,3 +68,89 @@ def normalize_data(data, zero_id = 0, one_id = 1):
     #remove calibration points
     norm_data = [d for ind, d in enumerate(norm_data) if metadata[ind] == max(metadata)]
     return norm_data
+
+def cal_scale(data):
+    """
+    Scale the data assuming 4 cal points
+
+    Parameters
+    ----------
+    data : unscaled data with cal points
+
+    Returns
+    -------
+    data : scaled data array
+    """
+    # assume with have 2 cal repeats
+    # TO-DO: make this general!!
```

Comment:
As a total aside, we should actually attempt this for multi-qubit data.

Reply:
not perfect, but I've been using this for any number of qubits: https://github.com/BBN-Q/Qlab.jl/commit/3732123def8903696796d466ada9dad765f55bb4